cmd/sebak/cmd/node.go

```diff
@@ -134,6 +137,8 @@ func init() {
 	nodeCmd.Flags().StringVar(&flagTLSCertFile, "tls-cert", flagTLSCertFile, "tls certificate file")
 	nodeCmd.Flags().StringVar(&flagTLSKeyFile, "tls-key", flagTLSKeyFile, "tls key file")
 	nodeCmd.Flags().Var(&flagValidators, "validator", "set validator: '<public address>,<endpoint url>,<alias>' or <public address>,<endpoint url>")
+	nodeCmd.Flags().StringVar(&flagSignThreshold, "sign-th", flagSignThreshold, "sign threshold")
```

Comment:
`sign-th` looks ambiguous, how about just using long expression such like `sign-threshold` and `accept-threshold` in below code?

Reply:
ok, seems better 👍